oauth2_http/java/com/google/auth/oauth2/DefaultCredentialsProvider.java

```diff
@@ -74,12 +72,13 @@ class DefaultCredentialsProvider {
   static final String CLOUDSDK_CLIENT_ID =
       "764086051850-6qr4p6gpi6hn506pt8ejuq83di341hur.apps.googleusercontent.com";
   static final String CLOUDSDK_CREDENTIALS_WARNING =
-      "Your application has authenticated using end user credentials from Google "
-          + "Cloud SDK. We recommend that most server applications use service accounts "
-          + "instead. If your application continues to use end user credentials from Cloud "
-          + "SDK, you might receive a \"quota exceeded\" or \"API not enabled\" error. For "
-          + "more information about service accounts, see "
-          + "https://cloud.google.com/docs/authentication/.";
+      "You are authenticating using user credentials. "
+          + "For production, we recommend using service account credentials.";
```

Comment:
WARNING: You are authenticating using user credentials. For production, we recommend using service account credentials.

To learn more about service account credentials, see http://cloud.google.com/docs/authentication/external/set-up-adc-on-cloud

Reply:
Thanks, did not notice the update